common/src/main/java/io/seata/common/loader/ExtensionDefinition.java

```diff
@@ -0,0 +1,101 @@
+/*
+ *  Copyright 1999-2019 Seata.io Group.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package io.seata.common.loader;
+
+import io.seata.common.util.StringUtils;
+
+/**
+ * The type URL
+ *
+ * @author haozhibei
+ */
+class ExtensionDefinition {
+    private String name;
+    private Class serviceClass;
+    private Integer order;
+    private Scope scope;
+
+    public Integer getOrder() {
+        return this.order;
+    }
+
+    public void setOrder(Integer order) {
+        this.order = order;
+    }
+
+    public Class getServiceClass() {
+        return this.serviceClass;
+    }
+
+    public void setServiceClass(Class clazz) {
+        this.serviceClass = clazz;
+    }
+
+    public Scope getScope() {
+        return this.scope;
+    }
+
+    public void setScope(Scope scope) {
+        this.scope = scope;
+    }
```

Comment:
I think all setters here are useless, should be removed.

Reply:
Removed